engine/src/main/java/org/terasology/rendering/opengl/FBOConfig.java

```diff
@@ -20,117 +20,125 @@
 import org.terasology.assets.ResourceUrn;
 
 /**
- * Builder class to simplify the syntax creating an FBO.
- * <p>
- * Once the desired characteristics of the FBO are set via the Builder's constructor and its
- * use*Buffer() methods, the build() method can be called for the actual FBO to be generated,
- * alongside the underlying FrameBuffer and its attachments on the GPU.
- * <p>
- * The new FBO is automatically registered with the LwjglRenderingProcess, overwriting any
- * existing FBO with the same title.
+ * An instance of this class uniquely describes an FBO.
+ *
+ * The description can then be used to create an FBO or to retrieve an FBO from one of the FBO managers.
+ *
+ * The constructors alone generate the description for two possible FBOs variety: one with no attached
+ * buffers and one with a single color buffer attached to it. To describe an FBO with additional buffers
+ * see the use*Buffer() methods.
```

Comment:
I updated all the javadocs here as the class had been cut&pasted from an FBOBuilder class and the javadocs where no longer up to date.